built-in-references/Kubernetes/container-no-privilege-escalation/constraint.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: constraints.gatekeeper.sh/v1beta1
```

Comment:
Before pushing policies into built-in-references folder here, similar to policies present in https://github.com/Azure/azure-policy-kubernetes/tree/prod/policies/GatekeeperV3, can you please write src.rego, src_test.rego to help understand the test coverage? Please check-in the policies along with tests first to azure-policy-Kubernetes repo or RegoPolicy repo.

Reply:
Sure, will push those up now then